### PR TITLE
Add Splash Screen

### PR DESCRIPTION
**Pull request by davidboll**
#### Description
Changes proposed in this pull request:
* Adds splash screen to Ionic App

#### What I have learned working on this feature: 
* Working with splash-screens and image-paths in ionic

#### Screenshots:
skarmavbild 2018-04-07 kl 23 34 42

### CODE REVIEW

File: src/app/app.component.ts [hunk 1]
```diff
@@ -11,12 +12,14 @@ import { TabsPage } from '../pages/tabs/tabs';
...
+  constructor(platform: Platform, statusBar: StatusBar, splashScreen: SplashScreen, modalCtrl: ModalController) {
     platform.ready().then(() => {
-      // Okay, so the platform is ready and our plugins are available.
-      // Here you can do any higher level native things you might need.
-      statusBar.styleDefault();
-      splashScreen.hide();
+      
+      let splash = modalCtrl.create(SplashPage);
+      splash.present();
+      // splashScreen.hide();
```
Comment (holgertidemand) [inline src/app/app.component.ts]: Do we use this? :)

**Review by holgertidemand — COMMENTED** [review]
Looks good David! 
Found a comment, but other then that, Nothing

File: src/app/app.component.ts (hunk 1 above)
Comment (davidboll) [reply src/app/app.component.ts]: Ah tnx! We do not, I'll take care of it

File: src/pages/splash/splash.scss [hunk 2]
```diff
@@ -0,0 +1,69 @@
+.ios, .md {
```
Comment (jyuw) [inline src/pages/splash/splash.scss:1]: Maybe have fewer line breaks in this file? :P

File: src/pages/splash/splash.ts [hunk 3]
```diff
@@ -0,0 +1,24 @@
+import { Component } from '@angular/core';
+import { ViewController } from 'ionic-angular';
+import { SplashScreen } from '@ionic-native/splash-screen';
+
```
Comment (jyuw) [inline src/pages/splash/splash.ts:4]: Here too plz

**Review by holgertidemand — APPROVED** [review]

**Pull request merged**